cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -79,14 +80,14 @@ type Planner struct {
 }
 
 // New creates a new Planner object.
-func New(autoscalingCtx *ca_context.AutoscalingContext, processors *processors.AutoscalingProcessors, deleteOptions options.NodeDeleteOptions, drainabilityRules rules.Rules) *Planner {
+func New(autoscalingCtx *ca_context.AutoscalingContext, processors *processors.AutoscalingProcessors, deleteOptions options.NodeDeleteOptions, drainabilityRules rules.Rules, nlt unneeded.LatencyTracker, ndt *deletiontracker.NodeDeletionTracker) *Planner {
```

Comment:
Please don't introduce `NodeDeletionTracker` to `Planner`. A snapshot of it is already passed to `UpdateClusterState` as `ActuationStatus`.